models/CoronSegmentator/scripts/cardiac_segmentation/cardiac_seg.py

```
@@ -0,0 +1,200 @@
+                futures = [
+                    executor.submit(
+                        self._process_single_segment, seg_image, segment, output_dir
+                    )
+                    for segment in self.segments
+                ]
+
+                for future in concurrent.futures.as_completed(futures):
+                    try:
+                        future.result(timeout=300)
+                    except TimeoutError as te:
+                        logger.error(f"TimeoutError: {str(te)}")
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)